nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -151,11 +151,13 @@ in
       };
 
       daemonIONiceLevel = mkOption {
-        type = types.int;
+        type = types.either types.int (types.enum ["idle"]);
```

Comment:
I heard what you're saying.  The problem is that "idle" supersedes the numeric priority, so then the documentation for the numeric priority setting would have to have a clause like "... unless idle=true, in which case this setting is ignored" ... which I think is *also* confusing.  :-/